design/components/cards.tsx

```diff
@@ -0,0 +1,89 @@
+import { useLink } from '@react-aria/link';
+import React, { useRef } from 'react';
+
+import { RichText } from './rich-text';
+import { styled } from './stitches.config';
+import { Tag } from './tag';
+import { Title } from './title';
+
+const StyledStories = styled('a', {
+  img: { height: 324, width: 406 },
+  position: 'relative',
+  outline: 'none',
+});
+
+const StyledCases = styled('a', {
+  img: { height: 324, width: 406 },
+  position: 'relative',
+  outline: 'none',
+});
```

Comment:
`StyledCases` and `StyledStories` is exactly the same.

Reply:
changed to be one